sdk/include/ndk/mmtypes.h

```diff
@@ -623,19 +623,6 @@ typedef struct _SEGMENT_OBJECT
     PMMSUBSECTION_FLAGS MmSubSectionFlags;
 } SEGMENT_OBJECT, *PSEGMENT_OBJECT;
 
-//
-// Section Object
-//
-typedef struct _SECTION_OBJECT
```

Comment:
According to my own research, that thing, like the SEGMENT_OBJECT either, is actually used just for dumping (partial) information in WinDbg when dumping section objects. So maybe you may keep it there, but add a comment about that (and saying that it does not represent the full actual SECTION object maintained by MM).

Reply:
According to this : 
https://www.vergiliusproject.com/kernels/x86/Windows%2010/1507%20Threshold%201/_SECTION_OBJECT
and that:
https://www.vergiliusproject.com/kernels/x86/Windows%2010/1511%20Threshold%202/_SECTION_OBJECT

SECTION_OBJECT disappeared somewhere near the infancy of Windows 10. ARM3 uses SECTION, which suit our needs better, and the two structures are not binary compatible.

Reply:
Actually to make my original point clearer, I was claiming that the SECTION_OBJECT was **never** used by the MM itself (even in WIn2003), but merely by the WinDbg dump extensions when they want to return a partial result of actual SECTION object. (By mapping some elements of an actual SECTION structure back into a SECTION_OBJECT for later presentation purposes.)